src/ensembl/src/content/app/entity-viewer/components/entity-viewer-topbar/scale-switcher/ScaleSwitcher.tsx

```diff
@@ -5,9 +5,15 @@ import styles from './ScaleSwitcher.scss';
 const ScaleSwitcher = () => {
   return (
     <div className={styles.container}>
-      This is placeholder for scaleswitcher
+      <div className={styles.emptyNode}></div>
+      <div className={styles.emptyNode}></div>
+      <div className={styles.emptyNode}></div>
+      <div className={styles.selectedNode}>Gene</div>
+      <div className={styles.emptyNode}></div>
+      <div className={styles.emptyNode}></div>
+      <div className={styles.emptyNode}></div>
```

Comment:
1) I think we prefer self-closing tags if they do not have any content:

`<div className={styles.emptyNode}></div>` -> `<div className={styles.emptyNode} />`

2) If we want to be slightly smarter about the repeated elements (not sure whether we want to, since this component will be changed significantly anyway), we could do something like this:

```
const renderEmptyDots = (number: number) => {
  return [...Array(number)].map((_, index) => <div key={index} className={styles.emptyNode} />;
}
```
and then:

```
<div className={styles.container}>
  {renderEmptyDots(3)}
  <div className={styles.selectedNode}>Gene</div>
  {renderEmptyDots(3)}
</div>
```